src/main/java/uk/gov/hmcts/sptribs/systemupdate/service/CcdSearchService.java

```diff
@@ -37,8 +37,6 @@ public class CcdSearchService {
     @Autowired
     private CoreCaseDataApi coreCaseDataApi;
 
-
-
     public List<CaseDetails> searchForAllCasesWithQuery(
         final BoolQueryBuilder query, User user, String serviceAuth, final State... states) {
 
```

Comment:
Method searchForCasesWithQuery can be converted to private and moved to last in the file.